app/javascript/components/Greeting.js

```diff
@@ -0,0 +1,19 @@
+import React, { useEffect } from "react";
+import { useSelector, useDispatch } from "react-redux";
+import { getGreetings } from "../controllers/greetingsSlice";
```

Comment:
- [ ] There is an import error here. You should import from `../redux/greetingsSlice`